docs/dev/ui-api-usage.md

```diff
@@ -0,0 +1,417 @@
+
+Response 
+
+Not sure whether or not this is used by the UI
+
+---
+
+# Policy Service
+
+https://github.com/eclipse-pass/pass-policy-service
+
+[Included docs](https://github.com/eclipse-pass/pass-policy-service/blob/main/web/README.md)
+
+Config: `POLICY_SERVICE_URL` base URL for the policy service
+
+## Policies
+
+Get a list of policies that apply to a submission, given its PASS ID (URI). Typically only used with in-progress submissions.
+
+|  |  |
+| --- | --- |
+| URL | `/policy-service/policies` |
+| Config | `POLICY_SERVICE_POLICY_ENDPOINT` |
+| Method | `GET` or `POST` |
+| Parameters | `submission` a submission ID |
+| Headers | `Content-Type: application/x-www-form-urlencoded` |
+| Body | `submission=submission_id` if a POST |
+| Response | An array of PASS Policy object IDs |
+
+Sample request:
+```
+GET /policy-service/policies?submission=<submission_id>
+```
+
+Sample response:
+``` JSON
+[
+ {
+   "id": "http://pass.local:8080/fcrepo/rest/policies/2d/...",
+   "type": "funder"
+ },
+ {
+   "id": "http://pass.local:8080/fcrepo/rest/policies/63/...",
+   "type": "institution"
+ }
+]
+```
+
+The `type` property identifies the source of the policy and will only take the values of `funder` or `institution`.
+
+## Repositories
+
+|  |  |
+| --- | --- |
+| URL | `policy-service/repositories` |
+| Config | `POLICY_SERVICE_REPOSITORY_ENDPOINT` |
+| Method | `GET` or `POST` |
+| Parameters | `submission` submission ID |
+| Headers | `Content-Type: application/x-www-form-urlencoded` |
+| Body | `submission=submission_id` if a POST |
+| Response | JSON object containing required, optional, and choice repositories |
+
+Sample request:
+```
+GET /policy-service/repositories
+```
+
+Sample response:
+``` JSON
+{
+	"required": [
+		{
+			"url": "http://pass.local/fcrepo/rest/repositories/1",
+			"selected": true
+		}
+	],
+	"one-of": [
+		[
+			{
+				"url": "http://pass.local/fcrepo/rest/repositories/2",
+				"selected": true
+			},
+			{
+				"url": "http://pass.local/fcrepo/rest/repositories/3",
+				"selected": false
+			}
+		],
+		[
+			{
+				"url": "http://pass.local/fcrepo/rest/repositories/4",
+				"selected": true
+			},
+			{
+				"url": "http://pass.local/fcrepo/rest/repositories/5",
+				"selected": false
+			}
+		]
+	],
+	"optional": [
+		{
+			"url": "http://pass.local/fcrepo/rest/repositories/6",
+			"selected": true
+		}
+	]
+}
+```
+* `selected` status denotes default choices, if the user is presented with options
+* `required` submissions MUST be depositied into these repositories
+* `one-of` array of arrays presenting choice-sets. The submission must be deposited into at least one from each choice-set. In this example, the submission must be deposited into repository (2 OR 3) AND (4 OR 5). 
+* `optional` the submission MAY be submitted to these repositories
+
+---
+
+# PASS Data Entities
+
+All CRUD requests for PASS data entities route through the [`pass-ember-adapter`](https://github.com/eclipse-pass/pass-ember-adapter).
```

Comment:
Do we want to list all the entities that are routed through the adapter for CRUD ops?